rero_ils/modules/loans/cli.py

```diff
@@ -38,10 +39,167 @@
 from ..notifications.api import Notification
 from ..notifications.tasks import create_notifications
 from ..patron_transaction_events.api import PatronTransactionEvent
+from ..patron_transactions.api import PatronTransaction
 from ..patron_types.api import PatronType
 from ..patrons.api import Patron, PatronsSearch
 from ..utils import extracted_data_from_ref, get_ref_for_pid, \
-    get_schema_for_resource
+    get_schema_for_resource, read_json_record
+
+
+def check_missing_fields(transaction, transaction_type):
+    """Return list of missing fields for a given transaction type.
+
+    transaction: the json transaction record.
```

Comment:
Should be: `:param`, see https://thomas-cokelaer.info/tutorials/sphinx/docstring_python.html for more details. Please check similar cases.